ui/src/main/webapp/src/app/components/reports/application-index/aggregated-statistics.service.ts

```diff
@@ -0,0 +1,35 @@
+import {Injectable} from "@angular/core";
+import {Http} from "@angular/http";
+import {Observable} from "rxjs";
+import {AbstractService} from "../../../services/abtract.service";
+import {Constants} from "../../../constants";
+import {IssueCategoryModel} from "../../../generated/tsModels/IssueCategoryModel";
+
+@Injectable()
+export class AggregatedStatisticsService extends AbstractService {
+    private static AGGREGATED_MANDATORY_INCIDENTS_URL = 'aggregatedstats/aggregatedMandatoryIncidents';
+    private static AGGREGATED_CATEGORIES_URL = 'aggregatedstats/aggregatedCategories';
+    private static AGGREGATED_JAVA_PACKAGES_URL = 'aggregatedstats/aggregatedJavaPackages';
+
+    constructor(private _http: Http) {
+        super();
+    }
+
+    getAggregatedCategories(executionId: number): Observable<any[]> {
```

Comment:
It might be better if these methods handled mapping the result data into typescript objects. It isn't clear to me what is being returned here.